docs/features/multiple-networks/multi-vtep.md

```diff
@@ -1 +1,200 @@
+"10.0.0.1,10.0.0.2,10.0.0.3"
+
+$ ovs-vsctl --timeout=15 --if-exists get Open_vSwitch . external_ids:ovn-pf-encap-ip-mapping
+"enp1s0f0:10.0.0.1,enp2s0f0:10.0.0.2,enp3s0f0:10.0.0.3"
+```
+With above `external_ids:ovn-encap-ip` settings, OVN creates 3 tunnel ports for
+each encap-ip on remote node, that's, if remote node has 2 encap-ips in
+`external_ids:ovn-encap-ip`, 6 tunel ports will be created on the local node.
+For example, if remote node only has one encap-ip "10.0.0.10", OVN creates
+below tunnel ports on the local node:
+```
+$ ovs-vsctl  show
+    Bridge br-int
+        Port ovn0-5e3e5-0
+            Interface ovn0-5e3e5-0
+                type: geneve
+                options: {csum="true", key=flow, local_ip="10.0.0.1", remote_ip="10.0.0.10", tos=inherit}
+        Port ovn0-e815c-1
+            Interface ovn0-e815c-1
+                type: geneve
+                options: {csum="true", key=flow, local_ip="10.0.0.2", remote_ip="10.0.0.10", tos=inherit}
+        Port ovn-33f2e5-0
+            Interface ovn-33f2e5-0
+                type: geneve
+                options: {csum="true", key=flow, local_ip="10.0.0.3", remote_ip="10.0.0.10", tos=inherit}
+```
+
+
+Ovn-controller programs physical flows that create mappings between
+<in_port, out_port> and <tunnel port> according to the aforementioned data
+model. The <in_port> information needs to be carried through a global register.
+**The bits 16-31 of the reg13 can be used for this purpose**. The reg13 is
+currently used to store conntrack zones for each VIF. However, it doesn’t
+really require 32 bits, and the first 16 bits (0-15) are sufficient. Its
+current usage with assignment of all 32 bits in certain pipelines need
+to be updated.
+
+When ovnkube-node adding a Pod's network interfaces to OVS br-int during the Pod
+creation, it identifies the corresponding PF interface for each VF interface, it
+then looks up `external_ids:ovn-pf-encap-ip-mapping` to determine the encap IP,
+which is subsequently assigned to the OVS Port's `external_ids:ovn-encap-ip`.
+This encap IP will be utilized as the source IP in the tunnel header.
+
+To steer outgoing traffic through the correct physical interface,
+the following source-based routing rules are required on k8s node:
+
+```
+$ ip rule list
+...
+6081:   from 10.0.0.1 lookup 6081 proto static
+6082:   from 10.0.0.2 lookup 6082 proto static
+6083:   from 10.0.0.3 lookup 6083 proto static
+
+$ ip route list table 6081
+10.0.0.0/23 dev vtep0 scope link
+10.0.0.0/16 via 10.0.0.1 dev vtep0 proto static metric 804
+$ ip route list table 6082
+10.0.0.0/23 dev vtep1 scope link
+10.0.0.0/16 via 10.0.0.1 dev vtep1 proto static metric 802
+$ ip route list table 6083
+10.0.0.0/23 dev vtep2 scope link
+10.0.0.0/16 via 10.0.0.1 dev vtep2 proto static metric 806
+```
+
+## Known Limitations
+
+This feature only works with network adapters that support SR-IOV,
+e.g. NVIDIA CONNECTX-6 or NVIDIA BlueField-2 in NIC mode.
```

Comment:
I think it works with any of the NICs, why only NVIDIA NICs?

Reply:
NVIDIA NICs are examples(`e.g.`)